app/api/v2/resources/auth.py

```diff
@@ -0,0 +1,272 @@
+# app/api/resources/auth.py
+"""Contains user logic"""
```

Comment:
This docstring is not needed, we talked about this

Reply:
Noted